code/modules/lore_codex/news_data/main.dm

```diff
@@ -1310,3 +1349,140 @@
+	<br><br>\
+	Military Liasion and long-standing veteran of the Moghes fleet Esksion Urahlia Kssrilan spoke this morning; \"To our fellow actors on the galactic stage. For long, we have been at odds with each other, in silent disdain over the space we both occupy. Today, we wish to set this aside. Today, we approach you out of honour, and our duty to our kin; as fellows. Word has spread that a vessel carrying our people, the HEG-TV Kinship of the Starchildren, has been seized by raiders from the criminal group known as Vystholm. The information we have gained suggests to us that our vessel, and its occupants - fathers, mothers, brothers and sisters, sons and daughters - will be taken to a sector known as Eutopia. A sector where forced labour thrives.\"\
+	<br><br>\
+	\"We shall never stand for such actions. None of ours shall be brought into slavery, to be forced into a duty designed by malice. Effective immediately, we will be moving vessels with peacekeepers and interventionalists of the clans Hisriar, Devorask and Oukiasto, to liberate anyone - your kin and our kin - from these chains. While history may still be fresh for some, we pray there is understanding for our intent. We ask nothing but your understanding, yet we will fully embrace any aid given.\"\
+	<br><br>\
+	\"May the Giver herself bless our steadfast warriors in their divine goal of liberation.\".\
+	<br><br>\
+	The action has been authorized by the SCG Fleet, who will be providing a diplomatic escort vessel, the SCG-D Kondiaronk to accompany the Hegemony fleet as far as the Gavel system where they are expected to depart alone for Eutopian territory. The precise composition and timeline for the operation remain classified, but a spokesperson for the Solar embassy expressed assurances that the action would be fruitful, and go far to mend relations with our unathi neighbours. The number of unathi vessels involved has not been specified, but civilian shipping and regional defense forces have been notified to remain calm and maintain contact with the Kondiaronk when hailed."
+
+/datum/lore/codex/page/article128
+	name = "08/04/75 - Vey-Medical Cruise Liner Evacuated After Accident In Vir"
+	data = "In what is being called one of the worst commercial transport incidents in Vir since the height of the Incursion, disaster struck the long-range cruise liner VTV Motes Dancing Across The Line Of Dawn yesterday. Reportedly, the Skrellian vessel was commencing a standard approach towards Kara via Sivian gravitational slingshot with around two thousand passengers and crew aboard, many travelling from as far as Qerr'balak, when a bluespace containment failure occurred, followed shortly by a skathari manifestation event. This resulted in an uncontrolled delamination, at least two fuel line ruptures, and depressurization of several decks, necessitating immediate evacuation.\

```

Comment:
```suggestion
	data = "In what is being called one of the worst commercial transport incidents in Vir since the height of the Incursion, disaster struck the long-range cruise liner VTV Motes Dancing Across The Line Of Dawn yesterday. Reportedly, the skrellian vessel was commencing a standard approach towards Kara via Sivian gravitational slingshot with around two thousand passengers and crew aboard, many travelling from as far as Qerr'balak, when a bluespace containment failure occurred, followed shortly by a skathari manifestation event. This resulted in an uncontrolled delamination, at least two fuel line ruptures, and depressurization of several decks, necessitating immediate evacuation.\

```